test/api_integration/apis/index_patterns/index_pattern_crud/create_index_pattern/main.ts

```diff
@@ -77,60 +76,85 @@ export default function ({ getService }: FtrProviderContext) {
       expect(response.body.index_pattern.sourceFilters[0].value).to.be('foo');
     });
 
-    it('can specify optional fields attribute when creating an index pattern', async () => {
-      const title = `foo-${Date.now()}-${Math.random()}*`;
-      const response = await supertest.post('/api/index_patterns/index_pattern').send({
-        index_pattern: {
-          title,
-          fields: {
-            foo: {
-              name: 'foo',
-              type: 'string',
-            },
-          },
-        },
+    describe('creating fields', () => {
+      before(async () => {
+        await esArchiver.load('index_patterns/basic_index');
       });
 
-      expect(response.status).to.be(200);
-      expect(response.body.index_pattern.title).to.be(title);
-      expect(response.body.index_pattern.fields.foo.name).to.be('foo');
```

Comment:
After the functional change this test stopped working because `foo` field was actually never saved into saved object